site/docs/v1/tech/apis/_tenant-response-body-base.adoc

```diff
@@ -62,12 +62,27 @@ The type of security protocol FusionAuth will use when connecting to the SMTP se
 [field]#{base_field_name}.emailConfiguration.setPasswordEmailTemplateId# [type]#[UUID]#::
 The Id of the Email Template that is used when a user had their account created for them and they must set their password manually and they are sent an email to set their password.
 
+[field]#tenant.emailConfiguration.unverified.allowEmailChangeWhenGated# [type]#[Boolean]# [since]#Available since 1.27.0#::
+When this value is set to `true`, the user is allowed to change their email address when they are gated because they haven't verified their email address.
+
+[field]#tenant.emailConfiguration.unverified.behavior# [type]#[String]# [since]#Available since 1.27.0#::
+The desired behavior during login for a user that does not have a verified email. The possible values are:
++
+* `Allow` - the user will be allowed to complete login.
+* `Gated` - verification is required before a user can complete login.
+
 [field]#{base_field_name}.emailConfiguration.username# [type]#[String]# [since]#Available since 1.8.0#::
 An optional username FusionAuth will to authenticate with the SMTP server.
 
 [field]#{base_field_name}.emailConfiguration.verificationEmailTemplateId# [type]#[UUID]#::
 The Id of the Email Template that is used to send the verification emails to users. These emails are used to verify that a user's email address is valid. If either the `verifyEmail` or `verifyEmailWhenChanged` fields are `true` this field is required
 
+[field]#tenant.emailConfiguration.verificationStrategy# [type]#[String]# [since]#Available since 1.27.0#::
```

Comment:
```suggestion
[field]#{base_field_name}.emailConfiguration.verificationStrategy# [type]#[String]# [since]#Available since 1.27.0#::
```

Reply:
This is needed because this include is used by both the tenant singular and plural response bodies.